html/inc/boinc_db.inc

```diff
@@ -57,31 +57,27 @@ class BoincDb extends DbConn {
             if ($host && $name && $user && $passwd) {
                 $retval = $instance->init_conn($user, $passwd, $host, $name);
                 if ($retval) {
-                    if ($instance->do_query("use $name")) {
-                        //error_log("BoincDb::get_aux(): connected to replica DB $dbnum");
-                        self::$instance = $instance;
-                        return;
-                    }
+                    //error_log("BoincDb::get_aux(): connected to replica DB $dbnum");
```

Comment:
Please remove dead code like this. You can add it back for debugging if needed. Alternatively, use pre-processor macros.

Reply:
Fixed by @bema-aei